### PR TITLE
Add E60 pcb

### PR DESCRIPTION
**Pull request by DigiD3ar**
## Description
add E60 pcb

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keeblounge/e60/readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# E60
+
+![E60](https://i.imgur.com/3IYT8Ne.jpg)
+
+A simple 5x12 pcb.
+
+* Keyboard Maintainer: [Digid3ar](https://github.com/DigiD3ar)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/keeblounge/e60/readme.md:9]: could you add instructions on how to enter the bootloader?

Comment (DigiD3ar) [reply keyboards/keeblounge/e60/readme.md:9]: added on last commit

**Review by drashna — APPROVED** [review]

File: keyboards/keeblounge/e60/info.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+    "keyboard_name": "E60",
+    "url": "https://github.com/DigiD3ar/e60",
+    "maintainer": "Keeblounge",
+    "width": 12,
+    "height": 5,
```
Comment (drashna) [inline keyboards/keeblounge/e60/info.json:6]:
```suggestion
```

File: keyboards/keeblounge/e60/info.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    "layouts": {
+        "LAYOUT_all": {
+            "key_count": 60,
```
Comment (drashna) [inline keyboards/keeblounge/e60/info.json:9]:
```suggestion
```

File: keyboards/keeblounge/e60/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#include "e60.h"
+
+#define _DEFAULT 0
```
Comment (drashna) [inline keyboards/keeblounge/e60/keymaps/via/keymap.c:20]:
```suggestion
```

File: keyboards/keeblounge/e60/rules.mk [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/keeblounge/e60/rules.mk:19]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keeblounge/e60/rules.mk [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/keeblounge/e60/rules.mk:16]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

File: keyboards/keeblounge/e60/rules.mk [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = atmega32u4
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```
Comment (fauxpark) [inline keyboards/keeblounge/e60/rules.mk:8]:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
```

File: keyboards/keeblounge/e60/rules.mk [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = atmega32u4
+BOOTLOADER = caterina
```
Comment (fauxpark) [inline keyboards/keeblounge/e60/rules.mk:3]:
```suggestion

# Bootloader selection
BOOTLOADER = caterina
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**